src/main/java/underdogs/devbie/auth/controller/interceptor/BearerAuthInterceptor.java

```diff
@@ -25,8 +26,15 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         if (interceptorValidator.isNotValid(handler)) {
             return true;
         }
+
         String token = authExtractor.extract(request, "bearer");
-        String userId = jwtTokenProvider.extractValidSubject(token);
+        Claims claims = jwtTokenProvider.extractValidSubject(token);
+        String userId = claims.get("userId").toString();
+        String role = claims.get("role").toString();
+
+        if (interceptorValidator.hasRoleAnnotation(handler)) {
+            interceptorValidator.validateRole(handler, role);
+        }
```

Comment:
```suggestion
        if (interceptorValidator.isValidRole(handler, role)) {
            return true;
        }
```
이런식으로 변경하면 위에서 사용한 if 문과 통일성 있게 더 깔끔해지지 않을까요?

Reply:
그렇네요. 말씀하신대로 수정했습니다! 👍